src/formio.component.ts

```diff
@@ -36,7 +36,7 @@ const _isEmpty = require('lodash/isEmpty');
     '<formio-alerts *ngIf="!this.options.disableAlerts" [alerts]="alerts"></formio-alerts>' +
     '<div #formio></div>' +
     '</div>',
-  styles: [require('./formio.component.scss').toString()],
```

Comment:
Has this been tested in an app like https://github.com/formio/angular-app-starterkit? I could have swore I tried this before and it did not bring in the templates when this module is included in another application.